### PR TITLE
Login/logout

### PR DESCRIPTION
**Pull request by araspaas**
Add flash message for a user who tries to log in with wrong info

- Add sad path test for a user who tries to log in with wrong info
- Add else statement to create action
------------------------------------------------
Add feature to display a flash message to a user who tries to login while logged in

- Add tests to display a flash message for a user that is already logged in and tries to login again
- add if logic to new action
------------------------------------------------
Add feature that a user can log out

- Add test that a user can logout
- Add flash message to notify a user that they logged out

### CODE REVIEW

**Review by Eternal-Flame085 — APPROVED** [review]
LGTM

File: spec/features/users/login_spec.rb [hunk 1]
```diff
@@ -78,5 +78,31 @@
...
+        click_link "Log In"
+      end
+
+      expect(current_path).to eq("/login")
+
+      fill_in :email, with: "funbucket13@gmail.com"
+      fill_in :password, with: fred.password
+      click_button "Log In"
+
+      expect(current_path).to eq("/login")
```
Comment (JBelta) [inline spec/features/users/login_spec.rb:104]: Gotta have the sad path :' (

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -10,6 +10,9 @@ def create
       session[:user_id] = user.id
       # flash[:login] = "Logged in as #{current_user.name}"
```
Comment (JBelta) [inline app/controllers/sessions_controller.rb]: Keep in nav?

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -1,7 +1,10 @@
 class SessionsController < ApplicationController
 
   def new
-
+    if session[:user_id] != nil
+      flash[:notice] = "You are already logged in!"
+      redirect_user
```
Comment (JBelta) [inline app/controllers/sessions_controller.rb:6]: good dynamic private method

**Pull request merged**